packages/block-editor/src/store/defaults.js

```diff
@@ -232,4 +232,214 @@ export const SETTINGS_DEFAULTS = {
 			slug: 'midnight',
 		},
 	],
+
+	// These are the same as the core theme.json.
+	// Added them here as well as per this conversation
+	// https://github.com/WordPress/gutenberg/pull/32482#discussion_r648956243
+	__experimentalFeatures: {
```

Comment:
What happens when a user does:

```
<BlockEditorProvider settings={{ __experimentalFeatures: { color: { custom: false } } } } />
```

Does it merge with the defaults, should we do that?